adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -108,7 +107,7 @@ void HybrisProximityAdaptor::sendInitialData()
            ProximityData *d = buffer->nextSlot();
 
            d->timestamp_ = Utils::getTimeStamp();
-           d->withinProximity_ = lastNearValue;
+           d->withinProximity = lastNearValue;
            d->value_ = lastNearValue ? 10 : 0;
```

Comment:
Actually... I can't find this "d->withinProximity", but there is withinProximity_  that is bool.

Which would mean lastNearValue == -1 -> withinProximity = true, value = 10 -> which is conflicting result, or?